shell/platform/darwin/ios/framework/Headers/FlutterPlugin.h

```diff
@@ -24,11 +24,19 @@ NS_ASSUME_NONNULL_BEGIN
 @protocol FlutterPlugin <NSObject>
 @required
 /**
- Registers this plugin.
+ Registers this plugin with a plugin registry.
+
+ The plugin registry itself is represented during registration by the provided
+ registrar, decoupling the registry API (managing a collection of registered
```

Comment:
If there's a relationship between the registry and the registration (e.g. the registry holds a number of registrations or outcomes of), perhaps point that out explicitly.